test/spec/unit/secureCreatives_spec.js

```diff
@@ -457,10 +458,13 @@ describe('secureCreatives', () => {
 
         sinon.assert.neverCalledWith(spyLogWarn, warning);
         sinon.assert.calledOnce(stubFireNativeTrackers);
-        sinon.assert.calledWith(stubEmit, CONSTANTS.EVENTS.BID_WON, adResponse);
-        sinon.assert.calledOnce(spyAddWinningBid);
+        sinon.assert.neverCalledWith(stubEmit, CONSTANTS.EVENTS.BID_WON);
+        sinon.assert.notCalled(spyAddWinningBid);
 
         expect(adResponse).to.have.property('status', CONSTANTS.BID_STATUS.RENDERED);
+
+        // revert back to 'adId = 1'
+        adResponse.adId = bidId;
```

Comment:
This should be in some setup / teardown section, or the next guy will also have to curse for however long it took you to track it down :)